doc/tutorials/stream_events.md

```diff
@@ -0,0 +1,297 @@
+
+Example:
+```js
+// listen to "streamEvent" events
+rxPlayer.addEventListener("streamEvent", (evt) => {
+  console.log("An event has been reached:", evt);
+});
+
+// listen to "streamEventSkipped" events
+rxPlayer.addEventListener("streamEventSkipped", (evt) => {
+  console.log("We just 'skipped' an event:", evt);
+});
+```
+
+
+
+## The event format ############################################################
+
+Whether you're listening to the `"streamEvent"` or the `"streamEventSkipped"`
+event, you will receive an object containing the corresponding event
+information.
+
+Here is an example of such events:
+```js
+{
+  start: 10, // start time of the event, in seconds.
+             //
+             // It is always defined, as a number.
+             //
+             // A start at `10` here means that the event began when the player
+             // reached the position at 10 seconds.
+
+  end: 25, // Optional end time of the event, in seconds.
+           //
+           // It can be undefined or unset for events without any duration.
+           // A end at `25` here indicates that this event only last from the
+           // position at 10 seconds (the `start`) to the position at 25
+           // seconds, or an event with a duration of 15 seconds.
+           //
+           // If `end` is defined, you can be notified when the end of this
+           // event is reached by adding an `onExit` callback to that event
+           // (continue reading this tutorial for more information).
+
+  data: { // The event's data itself.
+
+    type: EVENT_TYPE, // String describing the source of the event.
+
+    value: EVENT_VALUE, // This property's format and content depends on the
+                        // `type` property. For example, when the type property
+                        // is set to "dash-event-stream", this value will be the
+                        // <Event /> element corresponding to that DASH event.
+  }
+}
+```
+
+As written in this example, the underlying format of the event itself will
+depend on the source of the event. For example, an event generated from a DASH's
+`<EventStream />` won't be in the same format that an event generated from a
+MP4's `emsg` box.
+
+You can know which current format is used by checking the value of the
+`data.type` property.
+
+For now, we only have one format: DASH EventStream elements, which will have a
+`data.type` property equal to `"dash-event-stream"`.
+
+
+### DASH EventStream elements ##################################################
+
+A DASH EventStream's event will be parsed under the following format:
+
+```js
+{
+  start: 10, // As usual, the event start time in seconds
+
+  end: 15, // optional end position of the event, in seconds.
+           // Can be not set or set to `undefined` for events without a duration
+
+  data: {
+
+    type: "dash-event-stream", // Type corresponding to a DASH's EventStream's
+                               // Event element
+
+    value: {
```

Comment:
We know send the timescale too

Reply:
Yep did not take the time yet to update! (See comment just before)

Reply:
Done